tests/python_standalone_tests/test_neuron_build_and_sim_analytic.py

```diff
@@ -50,4 +50,4 @@ def test_python_standalone_neuron_build_and_sim_analytic(self):
 
         from nestmlmodule.test_python_standalone_module import TestSimulator
         neuron_log = TestSimulator().test_simulator()
-        np.testing.assert_allclose(neuron_log["V_abs"][-1], 11.192718053106296)
+        np.testing.assert_allclose(neuron_log["V_m"][-1], -58.80728194689356)
```

Comment:
Why did this change?

Reply:
Change from V_abs to V_m, change is exactly 70 mV.